modules/persistence/tests/metadata/metadata.test.ts

```diff
@@ -48,11 +48,6 @@ describe('metadata module', () => {
 
   describe('metadataFromZip', () => {
     it('should get metadata from site.bson', async () => {
-      const metaFromZip = await _metadataFromZip(zip);
-      expect(metaFromZip).toEqual({ ...meta, github_username: 'docs-builder-bot' });
-    });
-
-    it('should add github username to metadata', async () => {
```

Comment:
Consolidating these two tests since we now want to enforce a GitHub user before calling these functions

Reply:
oh i think that answers my question about throwing from index if no user(?)